packages/overlays/test/OverlayController.test.js

```diff
@@ -479,6 +485,13 @@ describe('OverlayController', () => {
 
         expect(ctrl.isShown).to.be.true;
 
+        // Don't hide on inside mousedown & outside mouseup
```

Comment:
Blocker:

I think we want to test here that:
- Does not hide on inside (content) mousedown ✅ 
- Does not hide on invoker mousedown <-- missing right now, this is the blocker

Nitpick:

We could remove the .click() checks because they wouldn't really have any effect now that we changed the event type.

Not sure where mouseup comes from, I don't see a reason to test that one but I also don't see the harm in it so we might as well, up to you!

Reply:
Added a test for this @jorenbroekema . You might want to have a look?
Then we can close this one